### PR TITLE
fix typo

### PR DESCRIPTION
**Pull request by vektor8891**
fix typo

### CODE REVIEW

File: src/shared/math/random.spec.ts [hunk 1]
```diff
@@ -38,18 +38,38 @@ describe('pseudo random generator', () => {
...
+        for (let i = 0; i < sampleSize; i++) {
+          const y = prng.randomInt(range)
+          res[y] = (res[y] ?? 0) + 1
+        }
+
+        expect(res.length).toEqual(range)
+        for (let i = 0; i < res.length; i++) {
+          res[i] = (res[i] ?? 0) / sampleSize
+        }
+        console.log(res)
```
Comment (maxinteger) [inline src/shared/math/random.spec.ts:57]: please remove the `console.log`

File: src/shared/math/random.spec.ts [hunk 2]
```diff
@@ -38,18 +38,38 @@ describe('pseudo random generator', () => {
...
       const res = [] as number[]
-      for (let i = 0; i < runCount; i++) {
-        const y = prng.randomInt(10)
-        res[y] = (res[y] ?? 0) + 1
-      }
 
-      expect(res.length).toEqual(10)
-      for (let i = 0; i < res.length; i++) {
-        res[i] = (res[i] ?? 0) / runCount
+      function checkDistribution(range: number) {
```
Comment (maxinteger) [inline src/shared/math/random.spec.ts:43]: next time please create a separated PR for changes which are not related to the original problem
In this case distribution check is not related to the typo

**Review by maxinteger — CHANGES_REQUESTED** [review]

File: src/shared/math/random.spec.ts (hunk 1 above)
Comment (vektor8891) [reply src/shared/math/random.spec.ts]: done.

File: src/shared/math/random.spec.ts (hunk 2 above)
Comment (vektor8891) [reply src/shared/math/random.spec.ts]: @maxinteger Apologies for the mess, these changes were not intentional. I removed them now. Can you pls have a look?

**Comment by vektor8891** [comment]
@maxinteger Apologies for the mess, these changes were not intentional. I removed them now. Can you pls have a look?

**Review by maxinteger — APPROVED** [review]

**Pull request merged**